timer/hrtimersnoop_example.txt

```diff
@@ -0,0 +1,18 @@
+Demonstration of hrtimersnoop.
```

Comment:
Also this should be a readme as per contributions guidelines:
https://github.com/bpftrace/user-tools/blob/master/CONTRIBUTING.md